src/index.js

```diff
@@ -84,17 +88,25 @@ class FBMessenger {
       cb = notificationType
       notificationType = this.notificationType
     }
+
+    const json = {
+      recipient: {
+        id: id
+      }
+    }
+
+    if (typeof data === 'string') {
+      json.sender_actions = data;
```

Comment:
This field must be written that way, and all the ones you point out below. Check the Send API for more information.

```
/* Send Request to set sender action to 'typing on'*/
curl -X POST -H "Content-Type: application/json" -d '{
  "recipient":{
  	"id":"USER_ID"
  },
  "sender_action":"typing_on"
}' "https://graph.facebook.com/v2.6/me/messages?access_token=PAGE_ACCESS_TOKEN"
```